azure-pipelines.yml

```diff
@@ -150,19 +150,15 @@ jobs:
     vmImage: ubuntu-18.04
   steps:
   - template: ci/install-package-dependencies.yml
-  - bash: |
-      set -x
-      sudo util/get-toolchain.py \
-        --install-dir="$TOOLCHAIN_PATH" \
-        --release-version="$TOOLCHAIN_VERSION" \
-        --update
-    displayName: Install toolchain
   - bash: |
       . util/build_consts.sh
-      ./meson_init.sh -A
-      ninja -C "$OBJ_DIR" \
-        sw/device/otp_img/otp_img_export_fpga_cw310 \
-        sw/device/lib/testing/test_rom/test_rom_export_fpga_cw310
+      ci/bazelisk.sh build \
+      //hw/ip/otp_ctrl/data:test_rma_image \
+      //sw/device/lib/testing/test_rom:test_rom_fpga_cw310_scr_vmem
```

Comment:
Nit: Please could you indent the continuation lines? That makes the file rather easier to read.

Reply:
yes